src/preset.ts

```diff
@@ -0,0 +1,86 @@
+const commonOptions: Fig.Option[] = [
+  {
+    name: "--no-interaction",
+    description: "Disable interactions",
+  },
+  {
+    name: "--debug",
+    description: "Display debug information instead of standard output",
+  },
+  {
+    name: "--silent",
+    description: "Do not print anything",
+  },
+];
+
+const applyOptions: Fig.Option[] = [
+  {
+    name: ["--path", "-p"],
+    description: "Path to a sub-directory in which to look for a preset",
+  },
+  {
+    name: ["--tag", "-t"],
+    description: "Branch or tag to use if the preset is a repository",
+  },
```

Comment:
I think this also takes an arg?
```suggestion
  {
    name: ["--tag", "-t"],
    description: "Branch or tag to use if the preset is a repository",
    args: {
      name: "tag",
    },
  },
```

Reply:
Good catch!